rules-reviewed/jboss-eap/weblogic/weblogic-webservices.windup.xml

```diff
@@ -0,0 +1,114 @@
+<?xml version="1.0"?>
+<ruleset xmlns="http://windup.jboss.org/schema/jboss-ruleset" id="weblogic-webservices">
+    <metadata>
+        <description>
+            This ruleset contains rules to detect WebLogic specific web services code, and to assist in the conversion of this code
+            to code that is compatible with the JBoss Enterprise Application Platform.
+        </description>
+        <dependencies>
+            <addon id="org.jboss.windup.rules,windup-rules-javaee,2.3.0.Final"/>
+            <addon id="org.jboss.windup.rules,windup-rules-java,2.3.0.Final"/>
+        </dependencies>
+        <sourceTechnology id="weblogic" versionRange="(1,)"/>
+        <targetTechnology id="eap" versionRange="[6,)"/>
+        <tags>
+            <tag>reviewed-2015-05-21</tag>
+        </tags>
+    </metadata>
+    <rules>
+        <rule id="weblogic-webservices-01000">
+            <when>
+                <javaclass references="weblogic.wsee.jaxrpc.ServiceImpl">
+                    <location>INHERITANCE</location>
+                </javaclass>
+            </when>
+            <perform>
+                <classification title="WebLogic Web Service Implementation" effort="1" severity="mandatory"/>
+                <hint title="WebLogic Service Implementation Class" severity="mandatory">
+                    <message>
+                        This stub will need to be regenerated using Apache CXF.
```

Comment:
Can this be explained a bit more? Not sure what this means. Does it warrant further explanation in the hint?